From f2e4487c7b8fd406e5ee69afaeb40c80f8ca0676 Mon Sep 17 00:00:00 2001
From: "Antoine Martin" <dev@ayakael.net>
Date: Tue, 18 Jan 2022 17:34:09 +0000
Subject: [PATCH 1/1] command-line-api_fix-musl-build

Command-line-api looks for Runtime nupkgs for non-musl systems,
causing errors as they are not provided by artifacts package.
This patch fixes this by making linux-nusl-x64 the only 
platform supported

---
 src/System.CommandLine.Suggest/dotnet-suggest.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/System.CommandLine.Suggest/dotnet-suggest.csproj b/src/System.CommandLine.Suggest/dotnet-suggest.csproj
index ae97dfe6..276f0158 100644
--- a/src/System.CommandLine.Suggest/dotnet-suggest.csproj
+++ b/src/System.CommandLine.Suggest/dotnet-suggest.csproj
@@ -6,7 +6,7 @@
     <PackAsTool>true</PackAsTool>
     <PackageId>dotnet-suggest</PackageId>
     <ToolCommandName>dotnet-suggest</ToolCommandName>
-    <PackAsToolShimRuntimeIdentifiers>win-x64;win-x86;osx-x64;linux-x64</PackAsToolShimRuntimeIdentifiers>
+    <PackAsToolShimRuntimeIdentifiers>linux-musl-x64</PackAsToolShimRuntimeIdentifiers>
     <PackagedShimOutputRootDirectory>$(OutputPath)</PackagedShimOutputRootDirectory>
 
     <DotnetSuggestBuildNumber>.1</DotnetSuggestBuildNumber>
-- 
2.34.1

